tests/helpers.py

```diff
@@ -1213,10 +1214,17 @@ def create_pods_parallel(pvc_list, namespace, interface):
     wait_time = 300
     with ThreadPoolExecutor() as executor:
         for pvc_obj in pvc_list:
-            future_pod_objs.append(executor.submit(
-                create_pod, interface_type=interface,
-                pvc_name=pvc_obj.name, do_reload=False, namespace=namespace)
-            )
+            if not raw_block_pv:
```

Comment:
I suggest in order not to call create_pod() with executor.submit twice,  to set pod_dict_path to None in case it's raw_block_pv

Reply:
ack